tests/e2e/high_availability_experimental/not_replicate_from_old_main.py

```diff
@@ -113,5 +115,144 @@ def retrieve_data_show_replicas():
     interactive_mg_runner.stop_all()
 
 
+def test_not_replicate_old_main_register_new_cluster(connection):
+    # Goal of this test is to check that although replica is registered in one cluster
+    # it can be re-registered to new cluster
+    # This flow checks if Registering replica is idempotent and that old main cannot talk to replica
+    # 1. We start all replicas and main in one cluster
+    # 2. Main from first cluster can see all replicas
+    # 3. We start all replicas and main in second cluster, by reusing one replica from first cluster
+    # 4. New main should see replica. Registration should pass (idempotent registration)
```

Comment:
why is this important to us? Idempotent registration on other cluster? I think we don't care about 2nd cluster...

Reply:
resolved on call